jenkins-scripts/dsl/ignition_collection.dsl

```diff
@@ -284,8 +285,8 @@ gz_collection_jobs =
         'ignition_utils1-install_bottle-homebrew-amd64',
         'sdformat-ci-sdformat13-focal-amd64',
         'sdformat-ci-sdformat13-homebrew-amd64',
-        'sdformat-ci-sdformat13-windows7-amd64',
-        'sdformat-install-sdformat12_pkg-focal-amd64'
+        'sdformat-install-sdformat12_pkg-focal-amd64',
+        'sdformat-sdf-13-win'
```

Comment:
While updating the buildfarmer dashboard I got this shortname for sdformat: sdformat-gz-13-win

Is this difference expected?

Reply:
sdformat is the weird guy in the gz collections since it never used ign/gz prefixes and goes into its own naming. With this PR I'm trying to get a consensus in the naming of the jobs. As described in the PR description:
```
$software_name-$branch_unversioned-$version-win
```

Reply:
Makes sense to me